README.md

```diff
@@ -8,7 +10,114 @@ O seu funcionamento é simples: a usuária escolhe um [tema](https://themes.gohu
 formato **markdown**, edita um arquivo de configuração `.toml` para promover customizações e voilà - 🐭🪄 a mágica
 gopher acontece!
 
-## Visão geral do projeto
+Table of contents
```

Comment:
@FabioRNobrega we use portuguese on purpose in order to have a friendlier document, could you adapt it?